.gitignore

```diff
@@ -27,3 +27,11 @@ obj/
 [Rr]elease*/
 _ReSharper*/
 [Tt]est[Rr]esult*
+/.vs/YamlDotNet.Editor/v15/Server/sqlite3/storage.ide-wal

```

Comment:
You should be able to simplify this to just `.vs/`

Reply:
Yes, you are right.